api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -560,6 +565,22 @@ def delete_venue(venue_id: int) -> utils.BackofficeResponse:
     return redirect(url_for("backoffice_web.pro.search_pro"), code=303)
 
 
+@venue_blueprint.route("/<int:venue_id>/fully-sync", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.MANAGE_PRO_ENTITY)
+def fully_sync_venue(venue_id: int) -> utils.BackofficeResponse:
+    venue = get_venue(venue_id)
+    has_active_provider = any(venue_provider.isActive for venue_provider in venue.venueProviders)
+
+    if not has_active_provider:
+        flash(f"Impossible de lancer la synchronisation du lieu {venue.common_name} ({venue.id})", "warning")
```

Comment:
Attention au risque de sécurité : si un acteur culturel saisis `<script>alert('hello!');</script>` comme nom d'usage du lieu, le JS se retrouve probablement exécuté. C'est un bon exercice de tester :slightly_smiling_face: . On peut s'en protéger en utilisant le module markupsafe. Par homogénéité avec le reste, cela donnerait : 
```suggestion
        flash(Markup("Impossible de lancer la synchronisation du lieu <b>{name}</b> ({venue_id})").format(name=venue.common_name, venue_id=venue_id), "warning")
```